Crud.py

```diff
@@ -6,15 +6,16 @@
 from sqlalchemy.orm import sessionmaker
 from datetime import date
 from ICD10csv import get_icd10_csv
+import pandas as pd
 
 engine = create_engine(DATABASE_URI,echo=True)
-"""
+
 def recreate_database():
     Base.metadata.drop_all(engine)
     Base.metadata.create_all(engine)
 
 recreate_database()
-"""
+
 Session = sessionmaker(bind=engine)
```

Comment:
won't there be any problems with session loosing connection to database? maybe try using scoped session